invesalius/gui/preferences.py

```diff
@@ -602,8 +602,8 @@ def __init__(self, parent, tracker, robot):
 
     def __bind_events(self):
         Publisher.subscribe(self.ShowParent, "Show preferences dialog")
-        Publisher.subscribe(self.OnRobotStatus, "Robot connection status")
-        Publisher.subscribe(self.OnTransformationMatrix, "Load robot transformation matrix")
+        Publisher.subscribe(self.OnRobotStatus, "Robot to Neuronavigation: Robot connection status")
+        Publisher.subscribe(self.OnTransformationMatrix, "Neuronavigation to Robot: Set robot transformation matrix")
```

Comment:
If this function "OnTransformationMatrix" is strictly related to the robot transformation matrix, it could also be renamed to "OnRobotTransformationMatrix"

Reply:
Thanks, done.